apps/users-service/src/handlers/fetch-me.ts

```diff
@@ -0,0 +1,57 @@
+import Boom from '@hapi/boom';
+import got from 'got';
+import Intercept from 'apr-intercept';
+import { config as authConfig } from '@asap-hub/auth';
+import { framework as lambda } from '@asap-hub/services-common';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+import Users from '../controllers/users';
+import * as auth0 from '../entities/auth0';
+
+const validateUser = async (
+  request: lambda.Request,
+): Promise<auth0.UserInfo> => {
+  const headers = request.headers as {
+    authorization: string;
+  };
+
+  if (!headers.authorization) {
+    throw Boom.unauthorized();
+  }
+
+  const [type, token] = headers.authorization.split(' ');
+
+  if (type.toLowerCase() !== 'bearer') {
+    throw Boom.forbidden();
```

Comment:
Afaik usually Unauthorized is for when you haven't provided a valid token, and Forbidden for when you have provided a valid token, but do not have permissions for the operation. So I think this should be unauthorized instead?

Reply:
Just revisited the HTTP codes spec. You're totally right! 👍